aws/internal/keyvaluetags/key_value_tags.go

```diff
@@ -50,6 +55,32 @@ func (tags KeyValueTags) IgnoreAws() KeyValueTags {
 	return result
 }
 
+// Merge calls keyvaluetags.Merge() on the given DefaultConfig.Tags, if any,
+// with KeyValueTags provided as an argument, overriding the value
+// of any tag with a matching key.
+func (config *DefaultConfig) Merge(tags KeyValueTags) KeyValueTags {
```

Comment:
Nit: Recommend adding "Tags" to the method name in case we ever need to merge `DefaultConfig`

```suggestion
func (config *DefaultConfig) MergeTags(tags KeyValueTags) KeyValueTags {
```